rosdep/python.yaml

```diff
@@ -5350,6 +5350,17 @@ python3-can:
   ubuntu:
     '*': [python3-can]
     xenial: null
+python3-catkin-lint:
+  debian:
+    '*': [python3-catkin-lint]
+    buster: null
+    jessie: null
+    stretch: null
+  fedora: [python3-catkin_lint]
```

Comment:
@cottsay the mix of dashes and underscores strikes me as odd here? Is the underscored package name a canonical, preferred, or optional spelling for python3-catkin-lint or are they distinct?

Also, since all of the web-based package indices for Fedora appear to be offline at the moment, is there something like https://manpages.debian.org/buster/devscripts/rmadison.1.en.html for Fedora repositories?

Reply:
I have never been using Fedora so I cannot help you with that. When submitting previous `rosdep` keys, I got a comment to add Fedora entries so for all my new keys I look up the Fedora packages.

Reply:
Yeah that's understandable. I appreciate the effort to make sure all our supported platforms are represented in your contributions. My comment above was specifically addressed to one of my colleagues whose experience with Fedora I hold in high esteem.

Reply:
The Fedora pattern is `python<pkgversion>-<modulename>` for Python packages, and underscores are allowed in package names if the upstream package "naturally contains them."

Pip and the Ubuntu debs seem to refer to this package as `catkin-lint`, but the repository, Python module, and all references in the `README.md` are to `catkin_lint`. Since the RPM package has nothing to do with Pip or the debs, I didn't really take them into account when naming the package, so it ended up with the underscore. Pretty well all of the other ROS python RPMs are named in this way.

Reply:
> is there something like ... for Fedora repositories?

Not that I'm aware of, unfortunately.